### ISSUE TITLE
Is pretrained model available for those three datasets?

### ISSUE THREAD

**Issue by Odaimoko** [Tharun24/MACH#3]

**Comment by Tharun24** [comment]
For each dataset, we have an ensemble of models. The file sizes of each of  them is substantial and hence we don't provide pre-trained models. However, I can provide models for the smaller wiki10-31K dataset on the cloud if you insist. May I know how you want to use them? For just evaluation or any other purpose?

**Comment by Odaimoko** [comment]
Wiki10-31k can be sufficient. I am working on a project to investigate how sota extreme classification methods perform on tail labels. It would be nice not to train models again :)
